src/styles/blocks/todoapp.scss

```diff
@@ -0,0 +1,166 @@
+.todoapp {
+  margin: $appSideMargin 20px;
+  color: #4d4d4d;
+
+  &__title {
+    margin: 0;
+
+    text-align: center;
+    font-size: 100px;
+    font-weight: 100;
+    color: rgba(175, 47, 47, 0.15);
```

Comment:
Consider using a variable for the color rgba(175, 47, 47, 0.15) to make it reusable and easy to maintain.